forward/forward.go

```diff
@@ -169,13 +172,6 @@ func (f *Forwarder) cloneRequest(req *http.Request, u *url.URL) (*http.Request,
 	outReq.URL.Scheme = "http"
```

Comment:
Oops this line of code will also break other schemes like `ws://`. We probably have other code that prevents websocket from working via Lantern proxy. Worth a new issue.